src/identity.js

```diff
@@ -1620,6 +1622,8 @@ export default function Identity(mpInstance) {
                             mpInstance._APIClient.prepareForwardingStats
                         );
                     }
+
+                    // TODO: This is a v1 method. We should deprecate
```

Comment:
Please instead make this a ticket and add the ticket instead of this description